source/game.c

```diff
@@ -2893,38 +2891,43 @@ static void jokers_sel_row_on_selection_changed(SelectionGrid *selection_grid,
 {
     if (prev_selection->y == row_idx)
     {
-        JokerObject* joker_object = list_get(jokers, prev_selection->x);
-        erase_price_under_sprite_object(joker_object->sprite_object);
-        sprite_object_set_focus(joker_object->sprite_object, false);
+        JokerObject* joker_object = (JokerObject*)list_get_at_idx(&_active_jokers_list, prev_selection->x);
+        if(joker_object != NULL) {
+            erase_price_under_sprite_object(joker_object->sprite_object);
+            sprite_object_set_focus(joker_object->sprite_object, false);
+        }
```

Comment:
```suggestion
        if (joker_object != NULL) 
        {
            erase_price_under_sprite_object(joker_object->sprite_object);
            sprite_object_set_focus(joker_object->sprite_object, false);
        }
```

Allman style, we should reformat everything and add clang enforcement.